langkit/support/langkit_support-token_data_handlers.ads

```diff
@@ -160,7 +162,12 @@ package Langkit_Support.Token_Data_Handlers is
       --  token, then the second entry stands for the trivia that come after
       --  the first token, and so on.
 
-      Symbols : Symbol_Table;
+      Symbols       : Symbol_Table;
+      --  Symbol table for this handler. Note that this can be shared accross
+      --  multiple Token_Data_Handlers.
+
+      Lines_Starts  : Index_Vectors.Vector;
+      --  Table keeping count of line starts and line endings
```

Comment:
This is not enough to understand how this data is encoded in the vector: could you add what the positive value at index `N` represents?